samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1317,15 +1300,15 @@ This is referred to as your self-signed CA certificate.
 
 To create your self-signed CA certificate:
 
-1. :ref:`Download and install <nrf_cloud_multi_service_install_nrf_utils>` the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repository.
-#. Use the nRF Cloud Utilities :file:`create_ca_cert.py` Python script to generate the certificate:
+1. :ref:`Install the nRF Cloud Utils <nrf_cloud_multi_service_install_nrf_utils>`.
+#. Use the nRF Cloud Utils :file:`create_ca_cert` script to generate the certificate:
 
    .. code-block:: console
 
-      python3 create_ca_cert.py -c US -f self_
+      create_ca_cert -c US -f self_
 
    Remember to set ``-c`` to your two-letter country code.
-   See the `Create CA Cert <nRF Cloud Utilities Create CA Cert_>`_ section in the nRF Cloud Utilities documentation for more details.
+   See the `Create CA Cert <nRF Cloud Utils Create CA Cert_>`_ section in the nRF Cloud Utils documentation for more details.
 
    You should now have the following three files:
 
```

Comment:
Line 1344 below:  You can be use them to sign as many device credentials as you need. ->  You can use them to sign as many device credentials as you need.